src/components/pages/Manage/Manage.tsx

```diff
@@ -1,29 +1,48 @@
 import { Tooltip } from '@src/components/data-display';
-import { useLatestANTVersion } from '@src/hooks/useANTVersions';
 import {
-  dispatchArNSUpdate,
-  useArNSState,
-  useGlobalState,
-  useModalState,
-  useWalletState,
-} from '@src/state';
+  useAntsForWallet,
+  useAntsRequireUpdate,
+} from '@src/hooks/useAntsForWallet';
+import { useArNSRecordsForWallet } from '@src/hooks/useArNSRecordsForWallet';
+import { useModalState, useWalletState } from '@src/state';
 import eventEmitter from '@src/utils/events';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 
-import { doAntsRequireUpdate } from '../../../utils';
 import DomainsTable from '../../data-display/tables/DomainsTable';
 import { RefreshIcon, SearchIcon } from '../../icons';
 import './styles.css';
 
 function Manage() {
-  const [{ arioProcessId, aoNetwork, hyperbeamUrl }] = useGlobalState();
-  const [{ loading: loadingArnsState, domains, ants }, dispatchArNSState] =
-    useArNSState();
-  const { data: antVersion } = useLatestANTVersion();
-  const antModuleId = antVersion?.moduleId ?? null;
   const [{ walletAddress }] = useWalletState();
   const [, dispatchModalState] = useModalState();
   const [search, setSearch] = useState<string>('');
+  const {
+    data: antData = {},
+    isLoading: isLoadingAnts,
+    isRefetching: isRefetchingAnts,
+    refetch: refetchAnts,
+  } = useAntsForWallet();
+  const {
+    data: domains = {},
+    isLoading: isLoadingDomains,
+    isRefetching: isRefetchingDomains,
+    refetch: refetchDomains,
+  } = useArNSRecordsForWallet();
+  const { ants: antsRequireUpdate, isLoading: isLoadingAntsRequireUpdate } =
```

Comment:
nit: antsRequireUpdate seems to indicate boolean but this is an array of process id's - `antsRequiringUpdate` might be higher signal here.